### ISSUE TITLE
Using phoenix html tag constructors in custom columns requires manual escaping.

### ISSUE THREAD

**Issue by sublimecoder** [smpallen99/ex_admin#156]
Using custom index columns requires you to manually escape the HTML using safe_to_string

```elixir
index do
column :name
column :subscription, fn(user) -> 
  Phoenix.HTML.safe_to_string(Phoenix.HTML.Tag.content_tag(:span, user.subscription_type, "data-id": user.id, class: "category"))
end
```
Either the docs should be updated to reflect this or it should be fixed so it automatically calls safe_to_string. Not sure what the correct answer is for this.

**Comment by OvermindDL1** [comment]
The Phoenix.HTML library follows the pattern that *all* input passed in is not safe and needs to be made safe unless explicitly marked as such.  That pattern should really be followed here.

**Comment by romul** [comment]
@OvermindDL1  You could use Xain tags in such the cases:
```elixir
index do
  column :name
  column :subscription, fn(user) -> 
    markup do
      span(user.subscription_type, "data-id": user.id, class: "category")
    end
  end
```

**Comment by sublimecoder** [comment]
@OvermindDL1 yes but the docs should reflect this when the examples show just calling Phoenix tags inside the column. So the examples do not work. Which is a problem. I spent 45 mins trying to figure out why nothing would return but a simple string would return fine and the docs did not help, nor did stack overflow. So either the documentation should be updated or this is a bug and it's not working the way the library author intended. Either way one needs to change right?

**Comment by smpallen99** [comment]
Agree, the docs need to be updated. I’m working on some updates in ex_admin right now. I’ll review the docs too.

Steve

**Comment by smpallen99** [comment]
Just tested it on master. No need to escape. the following works
```elixir
      column "Test 2", fn _x ->
        content_tag :li, "hello", style: "color: red"
      end
```

**Comment by smpallen99** [comment]
@romul no need for the `markup do` for the xain tag on master.
